ci/courtesy-push/courtesy-push.js

```diff
@@ -26,114 +30,175 @@ function formatDeps(depArr) {
         const name = depDetails[1];
         const version = depDetails[3];
         console.log(name + ' ' + version);
-        newDepsDict[name] = version;
+
+        if (name.includes('_')) {
+            const basicName = name.split('_')[0];
+            if (!deps[basicName]) deps[basicName] = {};
+            if (!deps[basicName].configs) deps[basicName].configs = [];
+
+            deps[basicName].configs.push({ name, version, depStr: newDep });
+        } else {
+            if (!deps[name]) deps[name] = {};
+
+            deps[name].name = name;
```

Comment:
> I think it would be better to declare a type of this rather than expando object

Was it necessary to define **deps** as a class?
Now I just added a **jsdoc** description of the type.